53/bryankimani/my_playlist.py

```diff
@@ -0,0 +1,62 @@
+import os
+import spotipy
+import spotipy.oauth2 as oauth2
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def print_header():
+    print("-------------------------------------")
+    print("         MY PLAYLIST TRACKS          ")
+    print("-------------------------------------")
+    print()
+
+
+def spotify_auth():
+    client_credentials_manager = oauth2.SpotifyClientCredentials(
+        client_id=os.getenv("CLIENT_ID"), client_secret=os.getenv("CLIENT_SECRET")
+    )
+    sp = spotipy.Spotify(client_credentials_manager=client_credentials_manager)
+    return sp
+
+
+def show_tracks(tracks):
+    for i, item in enumerate(tracks["items"]):
+        try:
+            track = item["track"]
+            print("   %d %32.32s %s" % (i, track["artists"][0]["name"], track["name"]))
+        except TypeError:
+            break
+
+
+def my_playlist():
+    username = input("Please enter your spotify username: ")
+    try:
+        playlists = spotify_auth().user_playlists(username)
+        for playlist in playlists["items"]:
```

Comment:
_flat is better than nested_, try making this try/exception smaller, by putting logic into helper functions